libc/CMakeLists.txt

```diff
@@ -465,6 +518,23 @@ add_library(oelibc STATIC
     ${MUSLSRC}/search/hsearch.c
     ${MUSLSRC}/search/insque.c
     ${MUSLSRC}/search/lsearch.c
+    ${MUSLSRC}/select/select.c
+    ${MUSLSRC}/select/poll.c
+    ${MUSLSRC}/linux/epoll.c
+    ${MUSLSRC}/signal/kill.c
+    #${MUSLSRC}/signal/signal.c
```

Comment:
I am hitting link error (after I turned on the MBEDTLS_NET_C).
I am curious why signal.c is commented out.  Is it going to come in the next installment?

Reply:
I will see if we can stub that out.

Reply:
I added stubs so that signal() and sigaction() are silently ignored and kill() panics.